Parsers/ASimAuthentication/Parsers/vimAuthenticationCiscoMeraki.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: ASIM Authentication parser for Cisco Meraki
-  Version: '0.1.2'
-  LastUpdated: Apr 09 2024
+  Version: '0.2.1'
+  LastUpdated: June 7 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
Done

Reply:
Done